docs/nebula-design-concepts.md

```diff
@@ -0,0 +1,216 @@
+# Data modeling and design concept
+------
+**Nebula Graph** is comprised of four components: storage layer, metadata service, computation layer and client console. In this documentation, we'll discuss data modeling and architecture of Nebula Graph.
+
+---
+## Directed property graph
+
+Modeling data with directed property graph, Nebula Graph is very easy to understand,
+i.e. Nebula Graph consists of two graph elements: vertex and edge. Each
+vertex represents an entity, and each edge represents how two vertices are associated.
+
+![image][example1]
+
+<!-- ### Vertex
+
+Vertices are typically used to store entities with properties. In the preceding example, vertices are **player** and **team**.
+
+### Edges
+
+Edges are used to connect vertices to one another and therefore provide a means
+of building your entities. Edges in Nebula Graph always have a type, a start (src)and an end vertex (dst), and a direction. It should be noted that there
+may be multiple edges of the same or different types at the same time between the
+start (src)and end vertex (dst). In the preceding example, edges are **serve**
+and **likeness**.
+
+### Properties
+
+Both vertices and edges can have properties, which are effectively key/value pairs. We call them **schema** in Nebula Graph. Like Mysql, Nebula Graph is a strong schema database. The name and data type of the properties are determined before the data is written. Vertex can have one or more properties, and edges can have properties too.
```

Comment:
... The definition of these properties (name, type, etc.) are **Schema** in the **Nebula Graph**

Strong schema => strong typed

This paragraph is not accurate. It omits the concept of tags. I'll given an example later

Reply:
Thanks, please show me your example.